mods/lib/importPatterns.js

```diff
@@ -9,7 +9,7 @@ const _ = require('lodash');
 
 const impName = name => `(:<${name}>[a-zA-Z\\.\\$]+?)`;
 const modulePathRegex = `(:<dotPath>[.]+)?${impName('module')}`;
-const multipleModulesRegex = '(:<modules>[a-zA-Z\\.\\$]+[,]+[\s]*[a-zA-Z\\.\\$,\\s]+)';
+const multipleModulesRegex = '(:<modules>[a-zA-Z\\.\\$]+[,]+[s]*[a-zA-Z\\.\\$,\\s]+)';
```

Comment:
nope, not looking for the `s` character, it should really be [\\s], I believe this is corrected on master